client/src/i18n/en-US/index.js

```diff
@@ -678,15 +684,6 @@ export default {
         close: "You are about to close this submission for review.",
         accept_as_final: "You are about to accept this submission as final.",
       },
-      action: {
-        accept_for_review: "Accept for Review",
-        cancel: "Cancel",
-        reject: "Reject",
-        request_resubmission: "Request Resubmission",
-        open: "Open for Review",
-        close: "Close for Review",
-        accept_as_final: "Accept as Final",
-      },
```

Comment:
The text on the SubmissionStatusChange dialog is now broken for most actions.
![image_2022-08-29_161342523](https://user-images.githubusercontent.com/108834493/187289777-e11ca5d9-b7bf-4f87-9de9-4b0474613396.png)

Reply:
It looks like this was [deleted by accident](https://github.com/MESH-Research/CCR/commit/288c144c30ec1ce33baae981277fbd218e98021c#diff-efcdb06f270138963722782624b81ae46fbbfb2147c277338b151bcfd1c4c021) today. I replaced it.